FarmSimulator1/src/UI/ConsoleUI.java

```diff
@@ -0,0 +1,157 @@
+package UI;
+
+import Domain.Sale;
+import Service.Service;
+import Validator.ValidationException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.*;
+
+/**
+ * This class is our Order list, the class will desplay the menu of options from 1 to 4 and will throw a message if
+ * you try to choose something that not exist.
+ */
+public class ConsoleUI {
+    private Service s;
+
+    public ConsoleUI(Service s) {
+        this.s = s;
+    }
+
+    public void run() {
+        while (true) {
+            displayMenu();
+            Integer cmd = getCommand();
+            if (cmd == -1)
+                continue;
+            switch (cmd) {
+                case 1:
+                    addProduct();
+                    break;
+                case 2:
+                    saleProduct();
+                    break;
+                case 3:
+                    salesFromDay();
+                    break;
+                case 4:
+                    return;
+                default:
+                    System.out.println("that option does not exist");
+            }
+        }
+    }
+
+    private void salesFromDay() {
+        Scanner si = new Scanner(System.in);
+        System.out.print("From what date do you want to see the sales?");
+        LocalDate day;
+        try{
+            System.out.println("(yyyy-MM-dd): ");
+            day = LocalDate.parse(si.nextLine(), DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+            for(Sale sa:s.salesFromDate(day)){
+                System.out.println(sa.getP().getId()+ " " + sa.getP().getName());
+            }
+
+        }
+        catch (DateTimeParseException e){
+            System.out.println("Errror formating data.");
```

Comment:
Error formating data is a bit confusing
A bit more exact where your code would fail is:
Error parsing input date

Also, it's usefull to do e.printStackTrace(); (when your user is not from the console... I get it now that the user might not want to see the stack trace.. but usually exceptions should log a stack trace to be able to find bugs)